src/utilities/API.jsx

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const db = axios.create({baseURL:"https://nc-news-waad.onrender.com/api"});
+
+export const getData = (searchUrl) => {
+       return db.get(searchUrl)
+        .then(({data}) => {
+            return data;
+         })
+    }
+
+export const showLoading = (isLoading) => {
+    if (isLoading) {
```

Comment:
hmmm this isn't displaying for me on the articles page

Reply:
Done